Gemfile.lock

```diff
@@ -283,7 +307,7 @@ DEPENDENCIES
   thin
 
 RUBY VERSION
-   ruby 2.3.3p222
+   ruby 2.5.5p157
 
 BUNDLED WITH
-   1.17.3
+   2.1.4
```

Comment:
Did we update the bundler on beta and prod? If I remember correctly due to limitation on m-web side, we still use bundler <2. Need to check with @fab-girard